app/services/work_observer.rb

```diff
@@ -43,7 +43,7 @@ def self.after_deposit_complete(work_version, _transition)
             mailer.deposited_email
           end
     job.deliver_later
-    mailer.globus_deposited_email.deliver_later if work_version.globus?
+    mailer.globus_deposited_email.deliver_later if work_version.globus_endpoint && Settings.notify_admin_list
   end
```

Comment:
reverting the change in #3057, which didn't actually do anything (and arguably should be left as it was)